pkg/service/storage/ucan_test.go

```diff
@@ -5,30 +5,43 @@ import (
 	"context"
 	"fmt"
 	"math/rand/v2"
+	"net/http"
+	"net/url"
 	"testing"
+	"time"
 
 	"github.com/multiformats/go-multihash"
 	"github.com/storacha/go-libstoracha/capabilities/assert"
 	"github.com/storacha/go-libstoracha/capabilities/blob"
+	"github.com/storacha/go-libstoracha/capabilities/replica"
 	"github.com/storacha/go-libstoracha/capabilities/types"
 	"github.com/storacha/go-ucanto/client"
+	"github.com/storacha/go-ucanto/core/car"
+	"github.com/storacha/go-ucanto/core/dag/blockstore"
 	"github.com/storacha/go-ucanto/core/delegation"
 	"github.com/storacha/go-ucanto/core/invocation"
+	"github.com/storacha/go-ucanto/core/message"
 	"github.com/storacha/go-ucanto/core/receipt"
 	"github.com/storacha/go-ucanto/core/result"
+	"github.com/storacha/go-ucanto/core/result/failure"
 	fdm "github.com/storacha/go-ucanto/core/result/failure/datamodel"
 	"github.com/storacha/go-ucanto/core/result/ok"
 	"github.com/storacha/go-ucanto/did"
 	"github.com/storacha/go-ucanto/ucan"
-	"github.com/storacha/storage/pkg/internal/testutil"
 	"github.com/stretchr/testify/require"
+
+	"github.com/storacha/storage/pkg/internal/testutil"
 )
 
 func TestServer(t *testing.T) {
 	ctx := context.Background()
 	svc, err := New(WithIdentity(testutil.Alice), WithLogLevel("*", "warn"))
 	require.NoError(t, err)
-	t.Cleanup(func() { svc.Close(ctx) })
+	err = svc.Startup()
```

Comment:
Previously we never stated the storage service, now that it's using a queue for replication, we must start it for the queue to run.